packages/stable/src/api/annotations/annotationsApi.ts

```diff
@@ -140,4 +143,32 @@ export class AnnotationsAPI extends BaseResourceAPI<AnnotationModel> {
   public update = (changes: AnnotationChangeById[]) => {
     return this.updateEndpoint(changes);
   };
+
+  /**
+   * [Reverse lookup resources](TODO: use API Doc link when ready)
+   * const assetQueryData = {
+   *   limit: -1,
+   *   filter: {
+   *     annotatedResourceType: 'file' as const,
+   *     annotationType: 'images.AssetLink',
+   *     data: {
+   *       { id: 123 }
+   *     }
+   *   }
+   * };
+   *
+   * const resourceIdsResponse = this._client.annotations.reverseLookup(assetQuerydata);
+   */
+  public reverseLookup = (
+    filter: AnnotationReverseLookupRequest
```

Comment:
is filter optional?

Reply:
No, I would expect the user to supply this for the request to make sense. I think we should wait with merging this until the API doc is ready, so that it's clear what the API looks like :P

Reply:
https://github.com/cognitedata/service-contracts/pull/2199 API doc PR here